guides/source/upgrading_ruby_on_rails.md

```diff
@@ -1356,6 +1356,18 @@ config.middleware.insert_before(Rack::Lock, ActionDispatch::BestStandardsSupport
 
 Also check your environment settings for `config.action_dispatch.best_standards_support` and remove it if present.
 
+* In Rails 4.0, a configuration option was added for controlling default HTTP headers.  This can be configured by setting the `config.action_dispatch.default_headers` in an environment file.  The defaults, in 4.0 are as follows:
```

Comment:
Nit: I think this is a little more concise
> Rails 4.0 allows configuration of HTTP headers by setting `config.action_dispatch.default_headers`. The defaults are as follows:
...

Reply:
Oh yes! That reads much better. 👍